action.yml

```diff
@@ -102,19 +105,25 @@ runs:
         DD_SET_TRACER_VERSION_JAVA: ${{ inputs.java-tracer-version }}
         DD_SET_TRACER_VERSION_JS: ${{ inputs.js-tracer-version }}
         DD_SET_TRACER_VERSION_PYTHON: ${{ inputs.python-tracer-version }}
+        DD_SET_TRACER_VERSION_RUBY: ${{ inputs.ruby-tracer-version }}
         DD_INSTRUMENTATION_BUILD_SYSTEM_JAVA: ${{ inputs.java-instrumented-build-system }}
-        INSTALLATION_SCRIPT_URL: https://install.datadoghq.com/scripts/install_test_visibility_v4.sh
-        INSTALLATION_SCRIPT_CHECKSUM: 41c0df6833bd371cbebf38767cf358c94a3fd1547056281e3ae9cade53fb39e2
+        INSTALLATION_SCRIPT_URL: https://raw.githubusercontent.com/DataDog/test-visibility-install-script/5a311f31200b149d1d1989ede0217146a097bac6/install_test_visibility.sh
```

Comment:
Shouldn't this update be published in install.dd instead?

Reply:
Looks like a temp workaround to test the changes with a version of the script that has not been released yet

Reply:
Yes, exactly: this branch is to test the approach in real GitHub Actions environment. It's a draft, so it's not ready for review just yet.